--- src/uGlobsPaths.pas	2013-02-08 18:39:43.000000000 +0400
+++ src/uGlobsPaths.pas.n	2013-02-10 16:24:56.738183711 +0400
@@ -3,9 +3,9 @@
 interface
 
 var
-  gpExePath : String = '';  // executable directory
+  gpExePath : String = '/usr/bin/';  // executable directory
   gpCfgDir : String = '';  // directory from which configuration files are used
-  gpGlobalCfgDir : String = '';  // config dir global for all user
+  gpGlobalCfgDir : String = '/usr/share/doublecmd/';  // config dir global for all user
   gpCmdLineCfgDir : String = ''; // config dir passed on the command line
   gpLngDir : String = '';  // path to language *.po files
   gpPixmapPath : String = '';  // path to pixmaps
@@ -29,7 +29,7 @@
   gpExePath := ExtractFilePath(TryReadAllLinks(ParamStrUTF8(0)));
   DCDebug('Executable directory: ', gpExePath);
   
-  gpGlobalCfgDir := gpExePath;
+  // gpGlobalCfgDir := gpExePath;
   if gpCmdLineCfgDir <> EmptyStr then
   begin
     if GetPathType(gpCmdLineCfgDir) <> ptAbsolute then
@@ -48,8 +48,8 @@
   end;
 
   gpCfgDir := IncludeTrailingPathDelimiter(gpCfgDir);
-  gpLngDir := gpExePath + 'language' + DirectorySeparator;
-  gpPixmapPath := gpExePath + 'pixmaps' + DirectorySeparator;
+  gpLngDir := gpGlobalCfgDir + 'language' + DirectorySeparator;
+  gpPixmapPath := gpGlobalCfgDir + 'pixmaps' + DirectorySeparator;
   gpCacheDir := GetAppCacheDir;
 
   // set up environment variables
